docs/aca/05-aca-dapr-pubsubapi/index.md

```diff
@@ -238,21 +250,21 @@ Content-Type: application/json
 }
 ```
 
-Keep eye on the terminal logs of the Backend background processor, you will see that the message is received and consumed by the action method `api/tasksnotifier/tasksaved` and an information message is logged in the terminal to indicate the processing of the message.
+Keep an eye on the terminal logs of the Backend background processor as you will see that the message is received and consumed by the action method `api/tasksnotifier/tasksaved` and an information message is logged in the terminal to indicate the processing of the message.
 
-**Hint:** You can use VS Code [Dapr Extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-dapr) to publish the message directly, it will be similar to the below image:
+**Hint:** You can use the VS Code [Dapr Extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-dapr) to publish the message directly. It will be similar to the below image:
 ![dapr-pub-sub-code-extension](../../assets/images/05-aca-dapr-pubsubapi/dapr-pub-sub-code-extension.jpg)
 
 ##### 6. Optional: Update VS Code tasks and launch configuration files
 
-If you have followed the steps in the [appendix](../../aca/20-appendix/01-run-debug-dapr-app-vscode.md) In order to be able to run the 3 services together and debug them in VS Code, we need to update the files `tasks.json` and `launch.json` to include the new service we have added, it will be similar to what we have done exactly in this post. You can use this file to update [tasks.json](https://github.com/Azure/aca-dotnet-workshop/blob/3dafcc7b291590177f7ce3a8a629079076149863/.vscode/tasks.json) and this file to update [launch.json](https://github.com/Azure/aca-dotnet-workshop/blob/3dafcc7b291590177f7ce3a8a629079076149863/.vscode/launch.json) files.
+If you have followed the steps in the [appendix](../../aca/20-appendix/01-run-debug-dapr-app-vscode.md) so far in order to be able to run the two services together (frontend and backend api) and debug them in VS Code, we need to update the files `tasks.json` and `launch.json` to include the new service we have added. You can use this file to update [tasks.json](https://github.com/Azure/aca-dotnet-workshop/blob/3dafcc7b291590177f7ce3a8a629079076149863/.vscode/tasks.json) and this file to update [launch.json](https://github.com/Azure/aca-dotnet-workshop/blob/3dafcc7b291590177f7ce3a8a629079076149863/.vscode/launch.json) files.
```

Comment:
three services together (frontend, backend api, and backend processor) and debug them in VS Code